config/database.yml

```diff
@@ -1,30 +1,23 @@
-
 default: &default
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-
-development:
-  <<: *default
-  adapter: postgresql
-  encoding: unicode
-  database: wbooks
-  host: localhost
-  username: wbooks-api
-  password: wbooks-api
-
-test:
-  <<: *default
   adapter: postgresql
   encoding: unicode
   database: wbooks
   host: localhost
   username: wbooks-api
   password: wbooks-api
 
-production:
-  <<: *default
-  adapter: postgresql
-  encoding: unicode
+common: &common
   database: <%= ENV["DATABASE_DB"] %>
   username: <%= ENV["DATABASE_USR"] %>
   password: <%= ENV["DATABASE_PSW"] %>
   url: <%= ENV["DATABASE_URL"] %>
+
+development:
+  <<: *default
```

Comment:
Use just common or default, using both is confusing. You can also opt to put your local database, user and password in your .env file instead and use only <%= ENV[" variables"] %> in your secrets file.

Reply:
Done